package.json

```diff
@@ -9,8 +9,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.13.2",
```

Comment:
Would prefer the modal component to be something more light weight like react-modal as well, instead of an entire component library like MUI for just this one modal.